polylogue/cli/attachments.py

```
@@ -0,0 +1,308 @@
+def run_attachments_cli(args: argparse.Namespace, env: CommandEnv) -> None:
+    sub = getattr(args, "attachments_cmd", None)
+    if sub == "stats":
+        _run_attachment_stats(args, env)
+    elif sub == "extract":
+        _run_attachment_extract(args, env)
+    else:
+        env.ui.console.print(f"[red]Unknown attachments subcommand: {sub}")
+        raise SystemExit(1)
+
+
+def _run_attachment_stats(args: argparse.Namespace, env: CommandEnv) -> None:
```

Ruff (0.15.9)

[warning] Too many return statements (8 > 6)

(PLR0911)